.github/workflows/compare_index_changes.py

```diff
@@ -24,31 +25,66 @@ def compare_index_changes(new_index, old_index):
     old_modules = old_index["index"]
 
     for module_name in new_modules:
-        if module_name not in old_modules:
-            continue
-
         new_module = new_modules[module_name]
-        old_module = old_modules[module_name]
-
-        # Handled by another GH Action
-        if "version" not in new_module:
-            continue
-        if "commit" not in new_module:
-            continue
-        if "version" not in old_module:
-            continue
-        if "commit" not in old_module:
-            continue
-
-        if new_module["version"] != old_module["version"]:
-            if new_module["commit"] == old_module["commit"]:
-                print("Error: Attribute 'version' was changed, but not 'commit'; in module '%s'" % module_name)
-                sys.exit(1)
-
-        if new_module["commit"] != old_module["commit"]:
-            if new_module["version"] == old_module["version"]:
-                print("Error: Attribute 'commit' was changed but not 'version'; in module '%s'" % module_name)
-                sys.exit(1)
+        old_module = old_modules.get(module_name)
+
+        # Any remote resources must be reachable
+        validate_reachable_resources(new_module, old_module, module_name)
+
+        if old_modules:
```

Comment:
Shouldn't this still be `if module_name in old_modules`?

Reply:
If you look at line 29, you will see that if `module_name` is not in `old_modules`, then old module will be `None`. Thus condition here will evaluate to false.

Reply:
But this says `old_modules` not `old_module`.

Reply:
You're right, my bad